pkg/webui/components/safe-inspector/index.js

```diff
@@ -54,6 +54,15 @@ const m = defineMessages({
   byteOrder: 'Switch byte order',
 })
 
+const MSB = 'msb'
+const LSB = 'lsb'
+const UINT32_T = 'uint32_t'
+const representationRotateMap = {
+  [MSB]: LSB,
+  [LSB]: UINT32_T,
+  [UINT32_T]: MSB,
+}
```

Comment:
This implementation is rather weird to me. I'd prefer having a regular array that is being cylced

```javascript
const representations = [MSB, LSB, UINT32_T]
```

The array should be composed based on the provided props (we could even use the array itself as a prop) and then keep track of the index via state. This will also be more flexible should we add more/other representations in the future.